Adds product filter and total orders to admin orders.

Alejok28 opened the pull request:
Fixes #3

Alejok28 committed:
Adds product filter and total orders to admin orders.
Fixes #3

juan267 commented on app/helpers/shoppe/application_helper.rb:
```diff
@@ -73,5 +73,33 @@ def page_entries_info(collection, options = {})
       info.html_safe
     end
+
+    def total_boxes(orders)
```
Estos metodos tienen cara de ir en archivo que se llame orders_helper.rb no en el application_helper.rb

juan267 commented on app/helpers/shoppe/application_helper.rb:
```diff
@@ -73,5 +73,33 @@ def page_entries_info(collection, options = {})
+
+    def total_boxes(orders)
+      boxes = 0
+      orders.size.times do |i|
```
en vez de usar `size.times` usa `each` es una forma de iterar mucho mas facil de leer y acorde con el estilo de programar en ruby

juan267 commented on app/helpers/shoppe/application_helper.rb:
```diff
@@ -73,5 +73,33 @@ def page_entries_info(collection, options = {})
+
+    def total_price(orders)
+      price = 0
+      orders.size.times do |i|
```
Aca puedes usar el metodo `reduce` para escibrir esto mejor. Algo como `orders.reduce(0) {|sum, order| sum += order.price }`

juan267 commented on db/seeds.rb:
```diff
@@ -94,7 +94,23 @@ def get_file(name, content_type = 'image/jpeg')
+  
+  product = Shoppe::Product.find( Faker::Number.between(1, 3))
+  quantity = Faker::Number.between(1, 4)
+  Shoppe::Order.last.order_items.add_item(producto, quantity)
```
aca la variable se llama `product` escribiste `producto` en la linea 115 por lo cual seguro se va a romper

Alejok28 committed:
Puts view methods to orders_helper

juan267 reviewed (approved):
Super mucho mejor, esos métodos del orders_helper.rb pueden estar aun mejor expresados con un uso mas amplio de métodos como `map` y `reduce` y varios métodos mas de la clase `Enumerable` de ruby para que le des una revisada. Pero como estan ahorita estan bien.

Alejok28 committed:
-integrates Ransack to orders date form for export as csv
-Changes date field style on the order search form
-Shows total values without paginate
